internal/datastore/mysql/watch.go

```diff
@@ -112,7 +114,9 @@ func (mds *Datastore) loadChanges(
 	}
 	defer common.LogOnError(ctx, rows.Close)
 
-	stagedChanges := common.NewChanges()
+	stagedChanges := common.NewChanges(func(r revision.Decimal) int64 {
```

Comment:
maybe we should have this function for comparison defined somewhere once? that way, its not getting redefined per call

Reply:
https://github.com/authzed/spicedb/pull/1039/commits/9f80ab3d645a81f9b49c9a7331c61dac48833172#diff-63655d1aac45a55ee99ed6cc29a10501150a30470bc2c600bc3b6355fd315f9eL117